Automattic/wp-calypso
# QR Code: Add config

#### Changes proposed in this Pull Request

Related: pdjUbD-db-p2

* This PR is the initial and doesn't implement the whole QR login flow. 

But just set the config and some of the skeleton components in order to make the work easier to review in the future.
A few different smaller PRs vs one big one. 


New QE Login page link for now.
![Screen_Shot_2022-05-02_at_16_40_21](https://user-images.githubusercontent.com/115071/166343119-e57b03e9-9054-45db-afe8-f73f02bc92e0.png)


New QR Login page for now.
![Screen Shot 2022-05-06 at 12 41 47](https://user-images.githubusercontent.com/115071/167206485-053a4b8e-f811-4489-a17b-950b78a59ebb.png)



#### Testing instructions
* Load this PR on your local development. or the calypso live link. 
* Visit /log-in/ - notice the "login via Mobile app" link.
* The link takes you to QR code page. 
* Visit /log-in/es ( Spanish version of the login page.

* Notice that the link takes you to spanish version of the QR code page. 

Visit different login pages. 

Jetpack. 
/log-in?client_id=69040&redirect_to=https%3A%2F%2Fpublic-api.wordpress.com%2Foauth2%2Fauthorize%3Fresponse_type%3Dtoken%26client_id%3D69040%26redirect_uri%3Dhttps%253A%252F%252Fcloud.jetpack.com%252Fconnect%252Foauth%252Ftoken%253Fnext%253D%25252F%26scope%3Dglobal%26blog_id%3D0

WooCommerce
/log-in?client_id=50916&redirect_to=https%3A%2F%2Fpublic-api.wordpress.com%2Foauth2%2Fauthorize%3Fresponse_type%3Dcode%26client_id%3D50916%26state%3D448630ff78a8154644042f841c51eb96fa72f7f4947afdbf68cb138f03a9502a%26redirect_uri%3Dhttps%253A%252F%252Fwoocommerce.com%252Fwc-api%252Fwpcom-signin%253Fnext%253Dmy-dashboard%26blog_id%3D0%26wpcom_connect%3D1%26wccom-from%3D%26calypso_env%3Dproduction

Akismet
/log-in?client_id=973&redirect_to=https%3A%2F%2Fpublic-api.wordpress.com%2Foauth2%2Fauthorize%3Fclient_id%3D973%26response_type%3Dcode%26blog_id%3D0%26state%3Dce16b788a5a5e12d5732faa57e51cf3fe01f2a141f1602f6accd6d0a7a59e3d8%26redirect_uri%3Dhttps%253A%252F%252Fakismet.com%252Faccount%252F%253Faction%253Drequest_access_token%26implicit%3Dtrue%26new-user%3D0

Notice that the pages look as expected. 
No Login via the Mobile App link. 

 In this PR we should see the new QR Code login link only on in local development.
